src/components/list/listItem/CollapseItem.tsx

```diff
@@ -102,4 +102,4 @@ const CollapseFrame: React.FC<CollapseFrameProps> = ({
   );
 };
 
-export default CollapseFrame;
+export default CollapseItem;
```

Comment:
you have "PlainCollapseList" in the other file, why this isn't "PlainCollapseItem" but just "CollapseItem" ? What happens if you have "CollapseItem" and "CollapseList", the word "plain" is really confusing to me.